utils/common/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -452,4 +453,22 @@ class ExtensionsTest : WordSpec({
             fileFromStr shouldBe aFile()
         }
     }
+
+    "URI.getQueryParameter" should {
+        "return the value of a query parameter" {
+            URI("https://oss-review-toolkit.org?key=value").getQueryParameterValue("key") shouldBe listOf("value")
+        }
+
+        "return all values of a query parameter" {
+            URI("https://oss-review-toolkit.org?key=value1,value2,value3").getQueryParameterValue("key") shouldBe
+                    listOf("value1", "value2", "value3")
+        }
+
+        "work with multiple query parameters" {
+            val uri = URI("https://oss-review-toolkit.org?key1=value1&key2=value2")
+
+            uri.getQueryParameterValue("key1") shouldBe listOf("value1")
+            uri.getQueryParameterValue("key2") shouldBe listOf("value2")
+        }
```

Comment:
Please add test cases for absent / empty keys, like in

    URI("https://oss-review-toolkit.org")
    URI("https://oss-review-toolkit.org?key=")